src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -183,20 +183,28 @@ public function getCookieJar()
     /**
      * Returns the current Crawler instance.
      *
-     * @return Crawler|null A Crawler instance
+     * @return Crawler A Crawler instance
      */
     public function getCrawler()
     {
+        if (null === $this->crawler) {
+            @trigger_error(sprintf('The %s() method calling before request() method is deprecated since 4.1 and will throw an exception in 5.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
`Calling the "%s()" method before the "request()" one is deprecated since Symfony 4.1 and will throw an exception in 5.0.`
(same below)

Reply:
Thanks for your help, fixed